test.js

```diff
@@ -0,0 +1,53 @@
+var Nuclear = require("./dist/nuclear"); // v1.1.1
```

Comment:
Did you mean to commit this file? If so, it should be moved to the `tests` directory.